cpp/src/barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp

```diff
@@ -56,19 +56,16 @@ class StandardCircuitConstructor : public CircuitConstructorBase<arithmetization
         // m           l       r       o        c
         create_poly_gate({ one_idx, one_idx, one_idx, 1, 1, 1, 1, -4 });
     };
-    // This constructor is needed to simplify switching between circuit constructor and composer
```

Comment:
Here Kesha is referring to the splitting work. This constructor shouldn't exist, and now it doesn't.